aten/src/ATen/miopen/Descriptors.h

```diff
@@ -111,9 +111,10 @@ struct ConvolutionDescriptor
                       &miopenCreateConvolutionDescriptor,
                       &miopenDestroyConvolutionDescriptor>
 {
-  void set(miopenDataType_t dataType, miopenConvolutionMode_t c_mode,  int dim, int* pad, int* stride, int * upscale /* aka dilation */, int groups) {
+  void set(miopenDataType_t dataType, miopenConvolutionMode_t c_mode,  int dim, int* pad, int* stride, int * upscale /* aka dilation */, int groups, bool deterministic) {
```

Comment:
Is this a public API that can be used in the wild already?

Reply:
Maybe provide a default value for the `deterministic` parameter for backward compatibility?
Otherwise looks fine.

Reply:
Yes this is a public, supported API.  It was introduced July 2022 so would have been part of ROCm 5.3, more than 2 releases ago.  We're now at ROCm 5.6.

Reply:
No need to provide a default value for deterministic.  All call sites for this `set` function have been updated to pass through the deterministic parameter.

Reply:
> Yes this is a public, supported API. It was introduced July 2022 so would have been part of ROCm 5.3, more than 2 releases ago. We're now at ROCm 5.6.

Before making non-backward compatible changes, usually at least one release when the old API is marked deprecated is needed.

Reply:
> No need to provide a default value for deterministic. All call sites for this `set` function have been updated to pass through the deterministic parameter.

I mean it can be used outside of PyTorch repo, can't it?